lib/azure/armrest/configuration.rb

```diff
@@ -215,6 +215,7 @@ def self.log
       # Sets the log to +output+, which can be a file or a handle.
```

Comment:
@djberg96 shall we change the getter `def self.log` to return the logger only, instead of getting the log file name in the first place?